turbo/jsonrpc/eth_receipts.go

```diff
@@ -606,3 +606,8 @@ func (i *MapTxNum2BlockNumIter) Next() (txNum, blockNum uint64, txIndex int, isF
 	isFinalTxn = txNum == i.maxTxNumInBlock
 	return
 }
+
+// GetReceipts Made getReceipts method public to use in sentry multi client
+func (api *BaseAPI) GetReceipts(ctx context.Context, tx kv.Tx, block *types.Block, senders []common.Address) (types.Receipts, error) {
```

Comment:
FYI: if you don't want make it public (it's fine for me) - you can introduce proxy-class. Example:
`turbo/jsonrpc/eth_system.go` has `func (b *GasPriceOracleBackend) GetReceipts`

Reply:
made public receiptsGenerator